virttest/arch.py

```diff
@@ -89,4 +89,8 @@ def get_kvm_module_list():
         host_cpu_type = cpu.get_cpu_vendor_name()
         return ["kvm", "kvm-%s" % host_cpu_type]
     elif ARCH in ('ppc64', 'ppc64le'):
+        return ["kvm", "kvm-hv"]
```

Comment:
Similar as x86, it's possible to detect the probe module dynamically, so do you want add it together? if no, let's leave a TODO comment for that, thanks!

Reply:
Since vt only support POWER8 and POWER9, and they all support kvm_hv, so I only add a comment here.